libfwupd/fwupd-enums.h

```diff
@@ -64,6 +64,28 @@ typedef enum {
 	FWUPD_TRUST_FLAG_LAST
 } FwupdTrustFlags;
 
+/**
+ * FwupdFeatureFlags:
+ * @FWUPD_FEATURE_FLAG_NONE:			No trust
+ * @FWUPD_FEATURE_FLAG_CAN_REPORT:		Can report successful update
+ * @FWUPD_FEATURE_FLAG_DETACH_TEXT:		Can show detach text
```

Comment:
How are detach text and post-update text different?

Reply:
the detach image is stored in a `<screenshot>` and is shown before the update is deployed, and the update image is stored in a `<custom>` section and shown after the update has successfully completed.

Reply:
Oh... like the 8bitdo thing.  It either needs more description or a better name then.  Maybe PRE-INSTALL text?

Reply:
>  Maybe PRE-INSTALL text

Well, at the moment gnome-software explicitly tests for:

    if (gs_app_get_kind (app) == AS_APP_KIND_FIRMWARE &&
        fwupd_device_has_flag (dev, FWUPD_DEVICE_FLAG_NEEDS_BOOTLOADER)) {

...so it really is detach specific. That said, I think *another* feature of pre-install could be added in the future if needed.

Reply:
OK